Dockerfile

```diff
@@ -39,10 +40,10 @@ RUN apt-get update && \
         php5-gmp \
         php5-intl \
         php5-mcrypt \
-        python \
+        python-all \
```

Comment:
Practically nothing. [`python-all`](http://packages.ubuntu.com/trusty/python-all) depends on `python` and `python2.7`, which is a [dependency](http://packages.ubuntu.com/trusty/python) for `python` itself. Same thing for `python3-all`.

Not having them installed results an error when building with `stdeb`, listing them as dependencies, though!